modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -273,49 +269,19 @@ def swagger_doc
 
       response '400', 'Bad Request' do
         let(:sc_uuid) { nil }
-
-        schema type: :object,
-               properties: {
-                 errors: {
-                   type: :array,
-                   items: {
-                     properties: {
-                       title: {
-                         type: 'string',
-                         example: 'Bad request'
-                       },
-                       detail: {
-                         type: 'string',
-                         example: 'Must supply a corresponding SC id in order to submit evidence'
-                       },
-                       code: {
-                         type: 'string',
-                         example: '400'
-                       },
-                       status: {
-                         type: 'string',
-                         example: '400'
-                       }
-                     }
-                   }
-                 }
-               }
-
+        schema '$ref' => '#/components/schemas/errorModel'
         it_behaves_like 'rswag example', desc: 'returns a 400 response', skip_match: true, scopes:
       end
 
       response '404', 'Associated Supplemental Claim not found' do
-        let(:sc_uuid) { nil }
+        let(:sc_uuid) { '00000000-0000-0000-0000-000000000000' }
```

Comment:
I changed this UUID because this endpoint will now return a 422 Validation error (like in the example below where `ssn` doesn't match) if the `sc_uuid` is blank, since it's now a required attribute in the JSON body schema.